lib/appRelationsDiscovery.js

```diff
@@ -0,0 +1,65 @@
+const {validate} = require('./utils');
+
+const supportedSyntax = ['default','mermaid'];
+
+// eslint-disable-next-line sonarjs/cognitive-complexity
+async function getRelations(asyncApiDocs,requestedSyntax = 'default') {
```

Comment:
I think that we should go with approach when second argument is a object with options like:

```suggestion
const defaultOptions = {
  syntax: 'default',
}

async function getRelations(asyncApiDocs, { syntax } = defaultOptions) {
```

It's good when will we expand the options :)